docs/docs/rest-api/public/api/v2/types/check.raml

```diff
@@ -0,0 +1,141 @@
+#%RAML 1.0 Library
+uses:
+  strings: stringTypes.raml
+  command: mesosCommand.raml
+  numbers: numberTypes.raml
+types:
+  HttpCheck:
+    type: object
+    properties:
+      endpoint:
+        type: strings.Name
+        description: |
+          The endpoint name to use.
```

Comment:
Could you give an example? I'm not sure what valid values are.

Reply:
the values are in the set of endpoints provided by the user... there isn't an easy way to give an example.